src/main/java/racing/Players.java

```diff
@@ -0,0 +1,44 @@
+package racing;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Players {
+    private List<Car> cars;
+
+    public Players(List<Car> cars) {
+        this.cars = cars;
+    }
+
+    public Players(String carNames) {
+        this.cars = insertCars(splitCarNames(carNames));
+    }
+
+    public String[] splitCarNames(String carNames) {
```

Comment:
```suggestion
    private String[] splitCarNames(String carNames) {
```

외부에서 알 필요가 없을 것 같습니다.

Reply:
done